test/distributed/fsdp/test_fsdp_uneven.py

```diff
@@ -25,35 +25,43 @@
 
 
 class TestUnevenParamShard(FSDPTest):
-    def _get_ref_results(self, model, input, my_lr):
+    def _get_ref_results(self, device, model, input, my_lr):
         with torch.no_grad():
+            if TEST_CUDA:
```

Comment:
Hello, this might optimized to pythonic expression, like `to_device = self.rank if TEST_CUDA else device` I think. :D

And I saw there's some repeating in this logic, extract a method might be an option to reduce duplication, and convenient for other device if they want to reuse test-cases as well. Thanks